### DISCUSSION TITLE
Improving UX for Credential Submission

### DISCUSSION THREAD

**Discussion by nkongsuwan** [WebOfTrust/signify-browser-extension#181] [Ideas]
Here are some improvements I would like to suggest.

## 1. Display only credentials that users control
Currently, the Polaris browser extension displays all credentials from `_client?.credentials().list()`, which includes all credentials chained to the credentials issued to the users. For example, the Polaris extension for an ECR person displays their ECR vLEI credentials as well as ECR-Auth, LE, and QVI vLEI credentials. 

I suggest the extension should only display credentials directly issued to the AIDs that the users control. Instead, there should be a button on displayed credentials to inspect their 

Note: Trying to log in with other credentials than ECR also gives an error `Signify Client not connected`.

## 2. Add buttons to inspect credential information
Currently, there is no way to inspect information on credentials displayed on the Polaris extension. For example, a user with multiple ECR vLEI credentials will not be able to distinguish these credentials from each other.  

I suggest adding a button to each displayed credential for the user to inspect its information, including all chaining credentials.

![Screenshot 2567-05-28 at 11 36 30](https://github.com/WebOfTrust/signify-browser-extension/assets/58167639/54bb5d97-8d67-4810-88db-6e6757660703)

**Comment by 2byrds** [comment]
Thank you @nkongsuwan some of this has been implemented (which creds are shown and the chaining) and we'll be adding more credential related views soon.
